ui/core/proto_utils/action_id.ts

```diff
@@ -563,20 +563,10 @@ idOverrides[ActionId.fromSpellId(37443).toProtoString()] = ActionId.fromItemId(3
 export const defaultTargetIcon = 'https://wow.zamimg.com/images/wow/icons/large/spell_shadow_metamorphosis.jpg';
 
 const petNameToActionId: Record<string, ActionId> = {
-	'Gnomish Flame Turret': ActionId.fromItemId(23841),
-	'Mirror Image': ActionId.fromSpellId(55342),
-	'Water Elemental': ActionId.fromSpellId(31687),
-	'Greater Fire Elemental': ActionId.fromSpellId(2894),
```

Comment:
I would leave this commented instead of deleted incase a new rune shows up for any of those pets